crates/web-sys/src/features/gen_AllowedBluetoothDevice.rs

```diff
@@ -39,6 +39,8 @@ impl AllowedBluetoothDevice {
     #[doc = ""]
     #[doc = "*This API requires the following crate features to be activated: `AllowedBluetoothDevice`*"]
     #[doc = ""]
+    #[doc = "The type is actually a union over some types and  can not yet be explained any better."]
```

Comment:
FWIW this doesn't seem too helpful IMO, it's not really adding much more than the documentation could already mention.